src/components/ProposalDetailPanel.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { DocumentTextIcon } from '@heroicons/react/24/solid';
+import {
+  Panel,
+  PanelBody,
+  PanelHeader,
+  PanelTag,
+  PanelTitle,
+  PanelTitleTags,
+  PanelTitleWrapper,
+} from './Panel';
+import { ProposalTable } from './ProposalTable';
+
+interface ProposalDetailPanelProps {
+  title: string | undefined;
+  applicant: string;
+  applicantId: string | undefined;
+  version: number;
+  values: {
+    fieldName: string,
+    position: number,
+    value: string,
+  }[],
+}
+
+const ProposalDetailPanel = ({
+  title,
+  applicant,
+  applicantId,
+  version,
+  values,
+}: ProposalDetailPanelProps) => (
+  <Panel>
+    <PanelHeader>
+      <PanelTitleWrapper>
+        <PanelTitle>
+          {applicant}
+        </PanelTitle>
+        <PanelTitleTags>
+          {applicantId && (
+            <PanelTag badge="EIN">
+              {applicantId}
+            </PanelTag>
+          )}
+          {title && (
+            <PanelTag icon={<DocumentTextIcon />}>
+              {title}
+            </PanelTag>
+          )}
+        </PanelTitleTags>
+      </PanelTitleWrapper>
```

Comment:
I'm pretty uncomfortable with how this incarnation makes the page feel like an Organization page, not a Proposal page. Which I understand is part of the intent, but I think it risks muddying the user's understanding of what entity they're actually looking at here.

For the demo purposes, let's just stick with it, but I think I'm going to take another pass at understanding the feedback and suggesting some more alternatives that don't relegate the proposal title itself to the panel tag area.

Reply:
(This is more of a musing than anything else, feel free to mark as resolved.)

Reply:
Ah, thank you for that thoughtfulness - I implemented the feedback I got without considering that! Would you like me to make an issue for following up on that?

Reply:
Jim and I had a meeting about the header today, and I'm trying out a few design ideas in Figma. I'll create an issue for it; what you have here can roll in without further edits.

Reply:
Oh, hah, I already did make an issue for it! https://github.com/PhilanthropyDataCommons/data-viewer/issues/54